### DISCUSSION TITLE
SP purchased product report

### DISCUSSION THREAD

**Discussion by cyncxl30** [amzn/ads-advanced-tools-docs#89] [Q&A]
The <img width="574" alt="微信图片_20230521173516" src="https://github.com/amzn/ads-advanced-tools-docs/assets/29911930/b7b98704-9b88-4e42-ada0-4350dcd9ba28">obtained from the API shows inconsistent data compared to the data downloaded from the seller backend. The data retrieved from the API is less than the data downloaded from the seller backend. The time period for both sets of data is in the month of April.

**Comment by elizafarley** [comment]
Hi @cyncxl30, when you say seller backend, are you referring to [Seller Central](https://sellercentral.amazon.com/)? Or data pulled using the [Selling Partner API](https://developer-docs.amazon.com/sp-api/)?

**Reply by cyncxl30** [reply]
Hi,yes.is Seller Central,The data retrieved from Selling Partner API and Seller Central is inconsistent.

**Comment by zyzjian** [comment]
hi, I have the same problem, have you found the solution

Looking forward to your reply
